File: minimum platforms best solution .py
#User function Template for python3
def minimumPlatform(n,arr,dep):
    
    temp = [0]*2400
    
    for i in range(n):
        temp[int(arr[i])] += 1
        temp[int(dep[i])+1] -= 1
    
    min_ = 0
    for i in range(1,2400):
        temp[i] += temp[i-1]
        if temp[i] > min_:
            min_ = temp[i]
    return min_
        
    

File: minimum platforms best solution.cpp
sort(arr,arr+n);
sort(dep,dep+n);
int platneed = 0;
int result=0,i=1,j=0;
if(n==1)
return 1;
while(i<n&&j<n) {="" if(arr[i]<="dep[j])" {="" platneed++;="" i++;="" if="" (platneed=""> result)
result = platneed;
}
else
{
platneed--;
j++;
}
}
return result>0?result+1:0;
}
